<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>693</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Type Your BTC Address</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
background-color: qlineargradient(spread:pad, x1:0.489, y1:0.0113636, x2:0.506, y2:1, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(30, 30, 30, 255));
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_btc_address">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_captcha">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>The account balance will be credited automatically to this bitcoin address when logging out:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_submit">
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: maroon;
font: 10pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_captcha">
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>If your session is lost your funds will be credited to this return address automatically after 24 hours.
</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Type the Code</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
